extensions/wikia/AdEngine/js/provider/directGpt.js

```diff
@@ -69,6 +69,8 @@ define('ext.wikia.adEngine.provider.directGpt', [
 				'TOP_RIGHT_BOXAD',
 				'LEFT_SKYSCRAPER_2',
 				'LEFT_SKYSCRAPER_3',
+				'INCONTENT_BOXAD_1',
+				'BOTTOM_LEADERBOARD',
```

Comment:
Our branch is on the top of [ADEN-4015](https://github.com/Wikia/app/pull/11625) branch. Those changes are needed for recovery in BTF slots.